### ISSUE TITLE
[A11y]: Implement Tag-Group items as list

### ISSUE THREAD

**Issue by liviakuenzli** [lyne-design-system/lyne-components#1839]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/lyne-design-system/lyne-components/blob/master/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/lyne-design-system/lyne-components/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/orgs/lyne-design-system/projects/4) for an issue that matches the one I want to file, without success.

### Request type

Request for enhancement of a component

### It affects the following packages

Components

### Functionality

According to Point Nr. 26 on the Accessibility Feedback List from Access for All (https://sbb.sharepoint.com/:x:/s/sbbch/ES-qWxpYlmJEvQhASdTstBUBlIQHBq9SxcFF39j4B4xugQ?wdLOR=cA959C989-1E6F-B340-B7EA-7ABD93031B69) the tag group items should be implemented as list.
